content/en/platform/corda/5.0/deploying-operating/topology/_index.md

```diff
@@ -9,4 +9,71 @@ menu:
     weight: 3000
 section_menu: corda5
 ---
-# Infrastructure Topology
\ No newline at end of file
+# Infrastructure Topology
+
+The following topic shows topology diagrams for both Corda in AWS and Corda in Azure.
+
+The size of the Kubernetes cluster required is dependent on the workload that the Corda cluster needs to handle. The Corda configuration section below gives some guidance on initial resource requests/limits to apply to the Corda workers which, when combined with the number of replicas for each worker type, can be used to estimate the total resources required in the Kubernetes cluster. The number of nodes in the cluster, and their distribution across availability zones, should take into account requirements for the availability of the solution.
```

Comment:
Should the second sentence cross-ref this? https://corda5.preview.docs.r3.com/en/platform/corda/5.0/deploying-operating/deployment/deploying.html#resource-requests-and-limits

Reply:
Yep - had added that after your review after talking with David.